features/networking/multus.feature

```diff
@@ -416,4 +416,107 @@ Feature: Multus-CNI related scenarios
     Then the output should contain "net1"
     And the output should contain "net2"
     And the output should contain 2 times:
-      | macvlan mode bridge |
+	    | macvlan mode bridge |
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24488
+  @admin
+  Scenario: Create pod with Multus bridge CNI plugin without vlan
+    # Make sure that the multus is enabled
```

Comment:
This should be supported from 4.2

Reply:
as discussed in other comment, removing version gates for now unless anybody has any reason to keep them